### ISSUE TITLE
doc generation expects only 1 slash in the api/kind string

### ISSUE THREAD

**Issue by garethahealy** [plexsystems/konstraint#4]
The doc generation expects there to be only 1 slash in the api/kind:
- https://github.com/plexsystems/konstraint/blob/c29f17b69b4b7d9c1b5cd3c2bd2e26c8bdac864c/internal/commands/document.go#L117-L125

But this is not true for most projects, i.e.: istio

Comment:
```
# Test
# @Kinds networking.istio.io/v1alpha3/VirtualService
```

Generates:
```
|API Groups|Kinds|Description|
|---|---|---|
|networking.istio.io|v1alpha3|Test|
```

**Comment by jpreese** [comment]
@garethahealy thanks for creating this issue. I am working on a fix right now and will push `v0.3.2` afterward.

**Comment by jpreese** [comment]
@garethahealy resolved in [v0.3.2](https://github.com/plexsystems/konstraint/releases/tag/v0.3.2)

https://github.com/plexsystems/konstraint/commit/d0bab6cb079b965834bb272ee3bc2dcf5aa620cd

Please let me know if this resolved the issue for you. Thanks again!
